sdl-core/src/main/scala/io/smartdatalake/statusinfo/StatusInfoRequestContext.scala

```diff
@@ -0,0 +1,17 @@
+package io.smartdatalake.statusinfo
+
+import javax.servlet.ServletContext
+import javax.servlet.http.HttpServletRequest
+import javax.ws.rs.core.Context
+
+trait StatusInfoRequestContext {
```

Comment:
Please add small scaldoc description for the class...

Reply:
I ended up simplifying the code and removing the class, thanks.